Instructor-Guide--Assignments--Marking--Grading-View.md

```diff
@@ -43,9 +44,50 @@ The grading bar, located at the top of the grading view page, provides some usef
 
+## Filter Submissions
+
+MarkUs allows you to filter and/or sort submissions based on specific criteria, such as marking state, tags, and other relevant attributes.
+To do this, click on the "Filter Submissions" button on the [Grading Bar](#grading-bar):
+
+![Filter Submissions button](images/filter-submissions-button.png)
+
+This will open up a Filter Submissions modal:
+
+![Filter Submissions](images/filter-submissions-modal.png)
+
+From here you may:
+
+- Order submissions (in ascending/descending order) by one of the following:
+    - Group Name (default).
+    - Submission Date.
+    - Total Mark.
+- Filter submissions by one or more of the following:
+    - Marking State (see [Marking State](Instructor-Guide--Assignments--Marking--Set-Up.md#marking-state)) : Filter all submissions with the specified marking state. Select one of the marking states from the dropdown:
+        - In Progress
+        - Completed
+        - Released
+        - Requested Review
+    - Tags assigned to the submission (see [Tags](#tags)) : You can select multiple tags to filter all submissions that have at least one of the specified tags associated with them.
+    - Section of the group's inviter.
+    - Graders assigned to grade the submission (only accessible by instructors) : You can select multiple graders to filter all submissions that have at least one of the specified graders assigned to them.
+    - Annotation (see [Annotations](#annotations)) : Text that must be present in an annotation for the submission.
+    - Total Mark : You can specify minimum and/or maximum total mark values to define a range and filter all submissions with total marks within the specified range (inclusive).
+    - Total Extra Marks : You can specify minimum and/or maximum total extra marks values to define a range and filter all submissions with total extra marks within the specified range (inclusive).
+    - Criteria (see [Criteria](Instructor-Guide--Assignments--Marking--Criteria.md)) :  You can filter student submissions based on their marks for different criteria. Follow these steps to apply the filter:
+        - Select a criterion from the dropdown and click the "Add Criterion" button. The criterion will show up on the list below.
+        - Specify min and/or max values to define a range for that criterion.
+        - Add more criteria if needed.
+
+All options will be restored to their default states when the "Clear All" button is clicked.
+After the modal is closed, submissions navigable using the "Next" and "Previous" buttons from the
+Grading Bar will be filtered and ordered according to the specified options.
+
+Note: All filters are applied using AND logic. This means that when multiple filters are applied, the submissions navigable must meet all the selected filters simultaneously.
```

Comment:
Remove the "All filter..." sentence. Just keep it as "Note: When multiple filters are applied..." Also reword the part after the comma to make it clearer. Brainstorm with other SDS members if you're stuck.